app/assets/stylesheets/groceries/_show.css.scss

```diff
@@ -97,22 +96,34 @@
     td:nth-child(6) {
       text-align: center;
       font-size: 1.2rem;
-    }
 
-    .dropdown {
       i {
-        background: $green;
-        padding: 6px 4px 4px 6px;
+        background: $light-gray;
+        padding: 4px 6px;
         border-radius: 20px;
         color: white;
 
-        &:hover { background: darken($green, 5%); }
+        &:hover {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5